zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, ParagraphRuntimeInfo> runtimeInfos;
```

Comment:
I see what you mean.

Because of Note object is passed to underlying NotebookRepo layer, and each NotebookRepo layer take care of serialization, we have no control of serializer in the each NotebookRepo implementation. So i think it's better have @transient for them.

What we can change is serializer on both REST and websocket end point.

websocket
https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java#L107

rest
https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/server/JsonResponse.java#L103

Will it be possible to have exclusion strategy on websocket and rest endpoint which ignore @transient for a field "runtimeInfos" ?